proto/rill/admin/v1/api.proto

```diff
@@ -1918,17 +1918,21 @@ message GetReportMetaRequest {
   string owner_id = 5;
   google.protobuf.Timestamp execution_time = 6;
   repeated string email_recipients = 7;
+  bool anon_recipients = 8;
+  string metrics_view = 9;
+  string explore = 10;
+  string canvas = 11;
   reserved 4;
 }
 
 message GetReportMetaResponse {
   message URLs {
     string open_url = 1;
     string export_url = 2;
-    string edit_url = 3;
+    string unsubscribe_url = 3;
```

Comment:
How will the report owner/admin open the report if they want to edit it? (E.g. change the frequency or the title)

Reply:
They will need to visit the reports page on UI. 

We would need to detect if user is a project or org admin or creator of the report and add edit link. Currently we just add edit link for all Rill users but it has an issue if the user is not part of project (or not admin) otherwise it will be broken experience for them. For this reasons I just removed the edit link and thought admin can visit the reports page, let me know if we want to add these checks.

Reply:
Though not great, this is alright. But would it be possible to keep the edit link for at least the owner user (I'm wondering if that would just require a simple check against the owner user ID)?

Reply:
Added edit link for owner, also added unsubscribe as a separate link since owner can be removed from project or org but should have option to unsubscribe.